src/static/js/modules/form-validation.js

```diff
@@ -6,34 +6,53 @@
 'use strict';
 
 var $ = require( 'jquery' );
+require( './jquery/cf_formValidator' ).init();
+require( './jquery/cf_notifier' ).init();
 
 function init() {
-  $( '.js-validate_form-not-empty' ).each( function() {
-    var $form = $( this );
-
-    $form.on( 'submit', function() {
-      var formIsEmpty = true;
-      var event;
-
-      $.each( $form.serializeArray(), function( index, element ) {
-        if ( element.value !== '' ) {
-          formIsEmpty = false;
-        }
-      } );
-
-      if ( formIsEmpty ) {
-        event = 'form:validate:empty';
-      } else {
-        event = 'form:validate:not_empty';
+  function sendNotification( elem, field ) {
+    var label = field.label;
+    var type;
+    var message;
+
+    for ( var prop in field.status ) {
+      if ( field.status[prop] === false ) {
+        type = prop;
       }
+    }
 
-      $form.trigger( event, $form );
+    if ( type === 'required' ) {
+      message = 'Please fill out the ' + label + ' field.';
+    }
 
-      return !formIsEmpty;
-    } );
+    if ( type === 'email' ) {
+      message = 'Please include a valid email address.';
+    }
+
+    if ( type === 'checkgroup' ) {
+      message = 'Please select at least one of the ' + label + ' options.';
+    }
 
-    return $form;
-  } );
+    $( elem ).trigger( 'cf_notifier:notify', {
+      message: message
+    } );
+    $( field.elem ).addClass( 'error' );
+  }
+
+  function clearNotification( elem ) {
+    $( elem ).trigger( 'cf_notifier:clear' );
```

Comment:
Consider creating a variable for $( elem ) so there isn't two subsequent DOM lookups.

Something like:
```js
  var $elem = $( elem );
  $elem.trigger( 'cf_notifier:clear' );
  $elem.find('.error').removeClass( 'error' );
```